portfolio/src/main/webapp/script.js

```diff
@@ -131,9 +131,9 @@ const fetchAndShowResponse = () => {
   } catch (err) {
     console.log(error);
   }
-  fetch('/data').then(response => response.text()).then(comment => {
-    commentPlaceholder.innerHTML = comment;
-  })
+  fetch('/data').then(response => response.text()).then(responseStrings => {
```

Comment:
On the client side how do you know that the response is a list of string?

Reply:
Ahh... Great point! I have updated it. Let me know if this is better.